### ISSUE TITLE
failes with upcoming flake8 3.8.0

### ISSUE THREAD

**Issue by fschulze** [tholo/pytest-flake8#66]
With the recently released flake8 3.8.0a2 the plugin fails:
```
lib/python2.7/site-packages/pytest_flake8.py:191: in check_file
    app.parse_preliminary_options_and_args(args)
AttributeError: 'Application' object has no attribute 'parse_preliminary_options_and_args'
```

**Comment by bobhy** [comment]
I have a V2 of this plugin which works (at least for me) with flake8 3.7 and 3.8.  It's clonable from: https://github.com/bobhy/pytest-flake8, be sure to checkout branch **use_flake8_config**.  Note, if you try this, it depends on flake8 config file almost exclusively.

**Comment by burritoIand** [comment]
could we update the title to include the error? was just pulling my hair out trying to figure out why my ci/cd just started failing out of the blue with this error :)

**Comment by noviluni** [comment]
I just added a PR that fixed this: https://github.com/tholo/pytest-flake8/pull/68

**Comment by MartinThoma** [comment]
I just got this error with `flake8-3.8.2` and `pytest-flake8==1.0.4`.

A work-around until this one is fixed seems to be to pin `flake8==3.7.9`

**Comment by FlorianHeigl** [comment]
@MartinThoma thank you so much, I got this while trying to fix a CI chain for an old project that I want to revive & port forward. I'm definitely not the right person to do that, but the one that does it. You saved me a lot of time now!
